src/member-api.js

```diff
@@ -51,19 +52,34 @@ export class MemberApi extends TypedEmitter {
    *
    * @param {Object} opts
    * @param {import('./capabilities.js').RoleId} opts.roleId
+   * @param {string} [opts.roleDescription]
    * @param {number} [opts.timeout]
    *
    * @returns {Promise<import('./generated/rpc.js').InviteResponse_Decision>}
    */
-  async invite(deviceId, { roleId, timeout }) {
-    const projectId = projectKeyToId(this.#projectKey)
+  async invite(deviceId, { roleId, roleDescription, timeout }) {
```

Comment:
think we should also limit the `roleId` to just member and coordinator, as it doesn't make sense to try to invite someone to be blocked, for example. not sure if best to update the param type or do a runtime check. ideally add a test for this too